pom.xml

```diff
@@ -623,7 +623,23 @@
 			<artifactId>directories</artifactId>
 			<version>26</version>
 		</dependency>
-	</dependencies>
+		<dependency>
+				<groupId>com.squareup.okhttp3</groupId>
+				<artifactId>okhttp</artifactId>
```

Comment:
We already do HTTP requests throughout UMS, why is this extra lib being added instead of doing it the same way we usually do?

Reply:
I also maintain the implementation in my control point at the moment. Switching implementation can be done, but makes more effort in keeping both implementation in sync. Maybe I'll habe a look and move to the jetty http client.

Reply:
That would be great. I like the idea of this change so I want to support it. I hope it wouldn't be a lot of work to use our http client instead

Reply:
No, it's not a lot of work. Have done it ...